### PR TITLE
fix: `after` rule when date is supplied

### PR DESCRIPTION
**Pull request by mankowitz**
### Description

What does this achieve?

This is a bug fix for when a number is provided to new date, see #647

Is it necessary to merge into core or can it be overridden using existing features?

Must be merged into core

### CODE REVIEW

File: resources/assets/js/helpers.js [hunk 1]
```diff
@@ -183,6 +183,11 @@ $.extend(true, laravelValidation, {
             var timeValue = false;
             var fmt = new DateFormatter();
 
+            if (typeof value === 'number' && format == undefined)
```
Comment (bytestream) [inline resources/assets/js/helpers.js:186]: Please can you make the latter check strict (`===`)

Comment (mankowitz) [reply resources/assets/js/helpers.js:186]: ok, done

File: resources/assets/js/helpers.js [hunk 2]
```diff
@@ -183,6 +183,11 @@ $.extend(true, laravelValidation, {
...
+            {
+                return value;
+            }
```
Comment (bytestream) [inline resources/assets/js/helpers.js:189]:
```suggestion
            if (typeof value === 'number' && typeof format === 'undefined') {
                return value;
            }
```

**Comment by bytestream** [comment]
Thanks @mankowitz

**Pull request merged**